specs/agents/otel-distribution.md

```diff
@@ -0,0 +1,77 @@
+
+## Terminology
+
+**Vanilla OpenTelemetry Distribution**: this is the "upstream" OpenTelemetry distribution that is maintained by the OpenTelemetry community.
+Implementation differs per platform, but it usually consists of an API/SDK and can also provide automatic instrumentation.
+
+**Elastic OpenTelemetry Distribution**: this is an OpenTelemetry distribution provided by Elastic that is derived from the 
+_Vanilla OpenTelemetry Distribution_.
+
+## General guidelines
+
+These statements are guiding principles of the Elastic OpenTelemetry distributions, they should be considered more as advice than strict rules.
+
+Elastic OpenTelemetry distribution SHOULD ideally:
+- behave as drop-in replacements of their upstream counterparts
+- provide a simple setup and favor onboarding experience (aka "things should work by default").
+- avoid capturing potentially confusing data (see [system metrics](#system-metrics) example below).
+
+## Configuration
+
+Elastic OpenTelemetry distributions MAY override the default configuration.
+When doing so, user-configuration should remain consistent with vanilla distribution:
+- explicit user configuration SHOULD remain effective
+- overriden default configuration MUST have the ability to be restored to upstream default
+
+Elastic specific configuration items MUST be prefixed with `ELASTIC_OTEL_`.
+For example, the [universal profiling integration](#universal-profiling-integration) can be enabled with `ELASTIC_OTEL_UNIVERSAL_PROFILING_INTEGRATION_ENABLED`.
+
+Elastic and platform specific configuration items must be prefixed with `ELASTIC_OTEL_${platform}_` to be consistent with
+the upstream `OTEL_${platform}_` prefix.
+
+For simplicity the configuration in this specification will use the "environment variable" syntax, some platforms like Java
+might also support other ways to configure.
+
+## Features
+
+### Inferred spans
+
+Supported platforms: [Java](https://github.com/elastic/elastic-otel-java/tree/main/inferred-spans)
+
+Configuration to enable: `ELASTIC_OTEL_INFERRED_SPANS_ENABLED`
+
+Note: While the implementation is Java-only for now, it should probably have been using `ELASTIC_OTEL_JAVA_INFERRED_SPANS_ENABLED`
```

Comment:
Shouldn't we instead first mention the config option name that should be used (`ELASTIC_OTEL_${platform}_INFERRED_SPANS_ENABLED`) and then put in the note that the current one (`ELASTIC_OTEL_INFERRED_SPANS_ENABLED`) will be updated?

Like 

```
### Inferred spans

Configuration to enable: `ELASTIC_OTEL_${platform}_INFERRED_SPANS_ENABLED`

Supported platforms: [Java](https://github.com/elastic/elastic-otel-java/tree/main/inferred-spans)

Note: While the implementation is Java-only for now, it currently uses the config option `ELASTIC_OTEL_INFERRED_SPANS_ENABLED `. It should be using `ELASTIC_OTEL_JAVA_INFERRED_SPANS_ENABLED` instead, but we plan to fix this inconsistency once it has been contributed upstream
```

Reply:
Emily, do you imply that we should use the language prefixed one if not every language implements that?

Reply:
Should be now clarified in 5d50ef97fa079a33d26c36165879ee3e4f1da171